src/NodeApi/JSProxy.cs

```diff
@@ -120,6 +120,21 @@ public Handler(string? name = null)
         /// </summary>
         internal JSObject JSHandler => (JSObject)JSHandlerReference.Value.GetValue()!;
 
+#if NETFRAMEWORK
+        public Apply? Apply { get; set; }
+        public Construct? Construct { get; set; }
+        public DefineProperty? DefineProperty { get; set; }
+        public DeleteProperty? DeleteProperty { get; set; }
+        public Get? Get { get; set; }
+        public GetOwnPropertyDescriptor? GetOwnPropertyDescriptor { get; set; }
+        public GetPrototypeOf? GetPrototypeOf { get; set; }
+        public Has? Has { get; set; }
+        public IsExtensible? IsExtensible { get; set; }
+        public OwnKeys? OwnKeys { get; set; }
+        public PreventExtensions? PreventExtensions { get; set; }
+        public Set? Set { get; set; }
+        public SetPrototypeOf? SetPrototypeOf { get; set; }
+#else
         public Apply? Apply { get; init; }
```

Comment:
>init [](http://example.com/codeflow?start=35&length=4)

We should also just add missing attributes? I wonder if we should be using the PolySharp project as a dependency. I guess it adds all such attributes for us.

Reply:
I did add some missing attributes... but the issue with `{ get; init; }` here is unrelated to attributes. .NET Framework runtime does not support it as far as I can see. The error is:

> CS0518 Predefined type 'System.Runtime.CompilerServices.IsExternalInit' is not defined or imported

Reply:
Here they say that we can just define the attribute ourselves to make it pass: https://youtu.be/WjTifkBlJHQ?t=170

Reply:
OK, it works. I was confused because it is not an **attribute**.